packages/codebytes/src/MonacoEditor/index.tsx

```diff
@@ -25,14 +25,11 @@ export const SimpleMonacoEditor: React.FC<SimpleMonacoEditorProps> = ({
   onChange,
 }) => {
   const editorRef = useRef<editor.IStandaloneCodeEditor | null>(null);
-  const editorWillMount = useCallback(
-    (editor: ThemedEditor, monaco: Monaco) => {
-      editorRef.current = editor;
-      monaco.editor.defineTheme('dark', dark);
-      monaco.editor.setTheme('dark');
-    },
-    []
-  );
+  const editorOnMount = useCallback((editor: ThemedEditor, monaco: Monaco) => {
+    editorRef.current = editor;
+    monaco.editor.defineTheme('dark', dark);
+    monaco.editor.setTheme('dark');
+  }, []);
```

Comment:
This allows us to set our own theme for the monaco editor. the reach team has so carefully manually crafted these theme colors in static sites, we're just seeing our own custom dark theme and sending it into the react monaco editor. 

there's more about this in the read me of the library: https://github.com/suren-atoyan/monaco-react#readme

<img width="935" alt="Screen Shot 2022-01-11 at 5 05 54 PM" src="https://user-images.githubusercontent.com/4481281/149028901-dd97ead5-1b21-470f-a7eb-4de03093fbe4.png">